demo/1-ui-deployment.yaml

```diff
@@ -13,8 +13,8 @@ spec:
         app: spring-boot-ui
     spec:
       containers:
-      - name: spring-boot-app-ui
-        image: <private docker repo>/ui:v2.0
+      - name: jackson-ui
+        image: mtarng/jackson-ui:v1.0
```

Comment:
Put in comment for people to replace this with their own UI image which points to their own cluster API endpoint

Reply:
With these changes, now the serves links that are relative paths. The routing is all handled through ingress gateway.

Reply:
How does the UI code resolve the address for `WEBPACK_PROP_API_BASE_URL` which is unique to the AKS cluster?

Reply:
Ah sorry - single public IP endpoint on Istio which serves up the UI and the APIs behind it.

Reply:
Yup. So we should only use the gateway. I removed the old load balancer services for api and the ui.